### ISSUE TITLE
Emilia fully annotated with EmoNet / Empathic Insight Voice emotion captions & scores - ready for training :)

### ISSUE THREAD

**Issue by christophschuhmann** [FunAudioLLM/CosyVoice#1465]
Hey, we ran our state of the art emotion captioning and emotion estimation scores over all of Emilia. You can find the results here distributed over these five repositories.

https://huggingface.co/datasets/laion/Emilia-with-Emotion-Annotations
https://huggingface.co/datasets/laion/Emilia-with-Emotion-Annotations2
https://huggingface.co/datasets/laion/Emilia-with-Emotion-Annotations3
https://huggingface.co/datasets/laion/Emilia-with-Emotion-Annotations4
https://huggingface.co/datasets/laion/Emilia-with-Emotion-Annotations5

https://huggingface.co/Orange/Speaker-wavLM-tbr

I would suggest fine-tuning that is also conditioned on these speaker embeddings because they capture the time-independent attributes of a voice that make up a speaker's identity without confusing it with emotions and arousal. This could eventually enable voice cloning without having a pair for the reference audio. Just take the embedding of the target as conditioning together with the emotion scores.

Have fun! :)

**Comment by zhuangweiji** [comment]
@christophschuhmann  Hi, thanks a lot for sharing these emotion annotations — awesome work!

I’d love to understand more about how the annotations were created:
- What model(s) did you use for the emotion captions and scores? What kind of data were they trained on?
- Any post-processing or filtering involved?
- How should we interpret the fields in the released datasets?

If there’s a blog post, paper, or doc you can share, that would be super helpful. Thanks again!

**Comment by christophschuhmann** [comment]
Sure, here are lots of informations:

[Do They See What We See? | LAION](https://laion.ai/blog/do-they-see-what-we-see/)


https://x.com/laion_ai/status/1935792645143494926  

https://arxiv.org/abs/2506.09827


We used this code:
https://github.com/LAION-AI/emotion-annotations

Here is a sheet for how to Interpret the scores:

Interpretation of Scores
The models predict raw scores. For the 40 Emotional Categories, these raw scores are also used to calculate a normalized Softmax Probability, indicating the relative likelihood of each emotion. Higher raw scores (shown in parentheses for emotions, or directly for attributes) generally suggest a stronger presence or intensity, aligning with the original annotation scales used during training.

Emotional Categories (40)
Original Annotation Scale: 0 (Not present at all) to 4 (Extremely present).
The table below displays: Softmax Probability (Raw Model Score)

Attribute Dimensions
Original Annotation Scale: Varies per dimension (detailed below).
The table for these dimensions displays: Raw Model Score

Details for Attribute Dimensions:
Valence: Range: -3 (Ext. Negative) to +3 (Ext. Positive). 0=Neutral.

Arousal: Range: 0 (Very Calm) to 4 (Very Excited). 2=Neutral.

Submissive vs. Dominant: Range: -3 (Ext. Submissive) to +3 (Ext. Dominant). 0=Neutral.

Age: Range: 0 (Infant/Toddler) to 6 (Very Old). (e.g., 2=Teenager, 4=Adult).

Gender: Range: -2 (Very Masculine) to +2 (Very Feminine). 0=Neutral/Unsure.

Serious vs. Humorous: Range: 0 (Very Serious) to 4 (Very Humorous). 2=Neutral.

Vulnerable vs. Emotionally Detached: Range: 0 (Very Vulnerable) to 4 (Very Detached). 2=Neutral.

Confident vs. Hesitant: Range: 0 (Very Confident) to 4 (Very Hesitant). 2=Neutral.

Warm vs. Cold: Range: -2 (Very Cold) to +2 (Very Warm). 0=Neutral.

Monotone vs. Expressive: Range: 0 (Very Monotone) to 4 (Very Expressive). 2=Neutral.

High-Pitched vs. Low-Pitched: Range: 0 (Very High-Pitched) to 4 (Very Low-Pitched). 2=Neutral.

Soft vs. Harsh: Range: -2 (Very Harsh) to +2 (Very Soft). 0=Neutral.

Authenticity: Range: 0 (Very Artificial) to 4 (Very Genuine). 2=Neutral.

Recording Quality: Range: 0 (Very Low) to 4 (Very High). 2=Decent.

Background Noise: Range: 0 (No Noise) to 3 (Intense Noise).


Best,
Christoph

**Comment by zhuangweiji** [comment]
@christophschuhmann 
Hi Christoph, thanks so much for the detailed explanation and links — super helpful!

I had a couple of follow-up questions:
- Is the 4500h training dataset for EmoNet-Voice Big open-sourced somewhere?
- Is EmoNet-Voice Foundation the same as EmoNet-Voice Big?
- I checked this link: https://huggingface.co/datasets/t1a5anu-anon/emonet-voice-foundation, but didn’t see any downloadable data — is it available elsewhere?
- Does this dataset contain EmoNet‑Voice Big? → https://huggingface.co/datasets/laion/laions_got_talent_enhanced_flash_annotations_and_long_captions

Appreciate your time, and thanks again for sharing all this great work!

**Comment by christophschuhmann** [comment]
We used these 2 datasets for training:
https://huggingface.co/datasets/laion/laions_got_talent_enhanced_flash_annotations_and_long_captions ( emonet big)

And these snippets additionally:
https://huggingface.co/datasets/mitermix/audiosnippets_long_2_8M

**Comment by ScottishFold007** [comment]
> https://huggingface.co/Orange/Speaker-wavLM-tbr

https://huggingface.co/Orange/Speaker-wavLM-tbr  The language support of this model is quite limited, isn't it? For example, it doesn't support Chinese, right?
